Rakefile

```diff
@@ -36,5 +37,5 @@ task :copy_man_page_to_manpath do |_t|
   end
 end
 
-task checks: [:rubocop, :rspec]
-task default: [:rubocop, :rspec]
+task checks: %i[rubocop rspec]
+task default: %i[rubocop rspec]
```

Comment:
Oddly, it failed on both `2.3` and `2.4`. Would this change be contributing to fixing these two versions? @olleolleolle